_stbt/core.py

```diff
@@ -1220,7 +1220,8 @@ def wait_for_motion(
     def ocr(self, frame=None, region=Region.ALL,
             mode=OcrMode.PAGE_SEGMENTATION_WITHOUT_OSD,
             lang=None, tesseract_config=None, tesseract_user_words=None,
-            tesseract_user_patterns=None, upsample=True, text_color=None):
+            tesseract_user_patterns=None, upsample=True, text_color=None,
+            text_color_threshold=25):
```

Comment:
If we're thinking that a different default might be a good idea perhaps this should be `=None` and we can look up the default with `get_config`?

Reply:
Good idea.